packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -988,22 +897,22 @@ export namespace EmrCreateCluster {
     /**
      * The statistic to apply to the metric associated with the alarm. The default is AVERAGE.
      *
-     * @default No statistic
+     * @default - None
      */
     readonly statistic?: CloudWatchAlarmStatistic;
 
     /**
      * The value against which the specified statistic is compared.
      *
-     * @default No threshold
+     * @default - None
      */
     readonly threshold?: number;
 
     /**
      * The unit of measure associated with the CloudWatch metric being watched. The value specified for Unit must correspond to the units
      * specified in the CloudWatch metric.
      *
-     * @default No unit
+     * @default - None
```

Comment:
What does it mean to have no units. That's what the default should describe.

Reply:
updated to the enum value that will be used (CloudWatchAlarmUnit.NONE)
I think the description of the behaviour belongs in the enum
unfortunately that's also documented as NONE and behaviour is not actually described.